src/Blocks/components/form/assets/form.js

```diff
@@ -504,26 +504,42 @@ export class Form {
 
 		if (this.state.getStateCaptchaIsEnterprise()) {
 			grecaptcha.enterprise.ready(async () => {
-				await grecaptcha.enterprise.execute(siteKey, {action: actionName}).then((token) => {
-					this.setFormDataCaptcha({
-						token,
-						isEnterprise: true,
-						action: actionName,
+				try {
+					await grecaptcha.enterprise.execute(siteKey, {action: actionName}).then((token) => {
```

Comment:
Either do `.then` or `await` but not both 😄 

So
a)
```js
 grecaptcha.enterprise.execute(siteKey, {action: actionName}).then((token) => { ... })
```

or
b)
```js
const token = await grecaptcha.enterprise.execute(siteKey, {action: actionName});
...
```

I recommend `async/await`, it's cleaner and easier to read.

Reply:
Added.